### PR TITLE
fix(spellchecker): Handle UTF-8 in misspelled words output

### PR DESCRIPTION
**Pull request by chriscrabtree**

### CODE REVIEW

**Comment by chriscrabtree** [comment]
@Baradoy please don't merge this yet. I have a better version almost ready!

**Comment by chriscrabtree** [comment]
@Baradoy Sorry for that--this approach is much cleaner and gives the plugin user better output even with invalid character data present in the misspellings. I'd be happy for you to merge this version now, if you find it acceptable. Thank you!

**Review by Baradoy — APPROVED** [review]
Thanks so much for this @chriscrabtree 

I will make a new release shortly.

**Pull request merged**
